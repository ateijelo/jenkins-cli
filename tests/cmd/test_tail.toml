bin.name = "jenkins"
args = ["tail", "-j", "hello", "-n", "1"]
