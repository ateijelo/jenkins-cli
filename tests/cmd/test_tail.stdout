hello #1: abcd
hello #1: efgh
